tools/dev/get_release_artifacts.py

```diff
@@ -0,0 +1,69 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import datetime
+import subprocess
+import shutil
+import sys
+
+from dateutil import parser, tz
+
+
+def main():
+    gh_bin = shutil.which("gh")
+    assert gh_bin, "Please install gh cli https://cli.github.com"
+    command = [
+        gh_bin,
+        "run",
+        "list",
+        "-w",
+        "release.yml",
+        "--json",
+        "status",
+        "--json",
+        "workflowName",
+        "--json",
+        "updatedAt",
+        "--json",
+        "databaseId",
+        "--json",
+        "conclusion",
+        "--json",
+        "headBranch",
+        "--json",
+        "headSha",
+        "--limit",
+        "10",
+    ]
+    output = subprocess.getoutput(" ".join(command))
+    artifact_builds = json.loads(output)
+
+    now = datetime.datetime.utcnow().replace(tzinfo=tz.tzutc())
+    candidate = None
+
+    for build in artifact_builds:
+        if not build['conclusion'] == 'success':
+            continue
+        if not build['headBranch'] == 'main':
+            continue
```

Comment:
Maybe clearer to add `--status success` and `--branch main` to the `gh run list` command?